### PR TITLE
All Assets need to maintain in database and lint fix

### PR DESCRIPTION
**Pull request by Manoradha77**

### CODE REVIEW

File: src/shared/function-helper.ts [hunk 1]
```diff
@@ -59,4 +59,6 @@ const downloadFunction = (url: string) => {
 
 const mobileViewValue = (state: RootState) => state.widthState.isMobileView;
 
-export { get, isEmpty, downloadFunction, getStatusMessage, mobileViewValue };
+const assets =(state: RootState) => state.assets.assetsData;
```
Comment (MJyotheesh) [inline src/shared/function-helper.ts:62]: format the code

File: src/shared/types.ts [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+
+declare global {
+  type ErrorType = Error | any;
+  type NotificationProps = {
+    notification: (type: string, message: string) => void;
+  };
+  type ObjectType = Record<string, string>;
+}
+
+export type StrObjectType = Record<string, string>;
```
Comment (MJyotheesh) [inline src/shared/types.ts:13]: ObjectType was already there right use that

File: src/redux/store.tsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import { configureStore, type ThunkAction, type Action } from "@reduxjs/toolkit";
-import {loginReducer,mobileViewReducer} from "./reduxSlice";
+import {loginReducer,mobileViewReducer, assetReducer } from "./reduxSlice";
```
Comment (MJyotheesh) [inline src/redux/store.tsx:2]: format the code

File: src/redux/reduxSlice.tsx [hunk 4]
```diff
@@ -17,9 +17,21 @@ const setMobileView = createSlice({
   reducers: {},
 });
 
+const setAssets = createSlice({
+  name: "assets",
+  initialState: { assetsData: [] },
```
Comment (MJyotheesh) [inline src/redux/reduxSlice.tsx:22]: make it object

File: src/pages/forgot-password/index.tsx [hunk 5]
```diff
@@ -167,100 +176,99 @@ const ForgotPassword = ({ notification }: forgotPasswordProps) => {
...
+          setError((prev) => ({
             ...prev,
             passWord: ENTER_YOUR_PASSWORD,
             confirmPassword: ENTER_CONFIRM_PASSWORD,
-          }));
+          })); 
         } else {
-          if (!formData.passWord) {
-            return setError((prev) => ({
+          if (formData.passWord == null) {
```
Comment (MJyotheesh) [inline src/pages/forgot-password/index.tsx:239]: change passWord to password whare and all we used

File: src/pages/forgot-password/index.tsx [hunk 6]
```diff
@@ -167,100 +176,99 @@ const ForgotPassword = ({ notification }: forgotPasswordProps) => {
...
+            setError((prev) => ({
               ...prev,
               passWord: ENTER_YOUR_PASSWORD,
               confirmPassword: "",
-            }));
-          } else if (!formData.confirmPassword) {
-            return setError((prev) => ({
+            })); 
+          } else if (formData.confirmPassword == null) {
```
Comment (MJyotheesh) [inline src/pages/forgot-password/index.tsx:245]: is this working properly?

**Review by MJyotheesh — CHANGES_REQUESTED** [review]
Resolve comments

File: src/shared/types.ts [hunk 7]
```diff
@@ -1,6 +1,26 @@
...
+
+export type vehicleDetails = Record<string, any>;
```
[the remark this answers is not in the document]
Comment (Manoradha77) [reply src/shared/types.ts:7]: If I remove this in types file then I want to change  this in vehicle details file (done by aravind have lint issues) so when he fix lint issues he will take care of this bro

[the remark this answers is not in the document]
Comment (MJyotheesh) [reply src/shared/types.ts:7]: okay

**Review by MJyotheesh — APPROVED** [review]

**Pull request merged**